src/pages/Lending/Pool/components/LoanSummary.tsx

```diff
@@ -160,6 +168,17 @@ export const LoanSummary: React.FC<LoanSummaryProps> = ({
   const { data: lendingQuoteCloseData, isLoading: isLendingQuoteCloseLoading } =
     useLendingQuoteCloseQuery(useLendingQuoteCloseQueryArgs)
 
+  const useRepaymentLockDataArgs = useMemo(
+    () => ({ assetId: collateralAssetId, accountId: collateralAccountId }),
+    [collateralAccountId, collateralAssetId],
+  )
+  const { data: positionRepaymentLock, isLoading: isPositionRepaymentLockLoading } =
+    useRepaymentLockData(useRepaymentLockDataArgs)
+
+  const useRepaymentLockNetworkDataArgs = useMemo(() => ({}), [])
```

Comment:
👀

Reply:
That's expected! Empty args to get the network repayment lock vs. user one